src/Pact/Analyze/Model/Text.hs

```diff
@@ -208,9 +210,10 @@ showEvent ksProvs tags event = do
               , emptyLine
               ]
         pure $ case scopeTy of
-          LetScope -> []
-          ObjectScope -> []
-          FunctionScope _ _ -> returnOutput
+          LetScope            -> []
+          ObjectScope         -> []
+          FunctionScope _ _   -> returnOutput
+          PactScope _ _       -> returnOutput
```

Comment:
I think maybe we don't want to return any output, because pacts are run for their side effects?